docs/weightonlyquant.md

```diff
@@ -186,12 +223,12 @@ tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
 prompt = "Once upon a time, there existed a little girl,"
 inputs = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
 
-qmodel = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True, device_map="xpu", trust_remote_code=True)
+qmodel = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True, device_map="xpu", trust_remote_code=True, use_llm_runtime=False)

```

Comment:
shall we need to expose this flag?

Reply:
doc has been updated by https://github.com/intel/intel-extension-for-transformers/pull/1356/ and https://github.com/intel/intel-extension-for-transformers/pull/1354, so close this old pr